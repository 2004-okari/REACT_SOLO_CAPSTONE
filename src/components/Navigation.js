import React from 'react'

function Navigation() {
  return (
    <>
      <h3 className='header-text'>Premier League</h3>
    </>
  )
}

export default Navigation