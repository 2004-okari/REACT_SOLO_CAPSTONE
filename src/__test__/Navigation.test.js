import React from 'react';
import renderer from 'react-test-renderer';
import Navigation from '../components/Navigation';

describe('Navigation displays correctly', () => {
  test('Navigation file matches snapshot', () => {
    const tree = renderer.create(<Navigation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
