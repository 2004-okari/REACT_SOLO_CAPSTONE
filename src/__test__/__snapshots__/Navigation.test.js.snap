// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation displays correctly Navigation file matches snapshot 1`] = `
<h1
  className="header-text"
>
  Premier League
</h1>
`;
